Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -605,4 +606,27 @@ private function getFinalControlContinuationTokensForOpeningToken($openingTokenK
 
         return array();
     }
+
+    private function fixDeclareStrictTypes(Tokens $tokens)
+    {
+        $declares = $tokens->findGivenKind(T_DECLARE);
+
+        foreach ($declares as $index => $declare) {
+            if ($this->isStrictTypeDeclare($index, $tokens)) {
```

Comment:
@keradus do you meant that there's no case for the block element formatting? In the PHP docs you can see that `ticks` can be used with curly bracers to apply to only a part of the file (which is invalid for `strict_types`) - http://php.net/manual/en/control-structures.declare.php#control-structures.declare

```php
<?php
// these are the same:

// you can use this:
declare(ticks=1) {
    // entire script here
}

// or you can use this:
declare(ticks=1);
// entire script here
?>
```

Since it's a control structure then I would assume it'd follow the same rules as the other control structure items in PSR 2 - https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#54-for - with the closest similar examle being the `for` section.

It could be the case that all single line `declare` statements are formatted the same (without spaces) but I couldn't find anything else mentioning this so I thought it best to just stick with what I could tell.